iot-hub/Samples/service/CleanupDevicesSample/Parameters.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+
+namespace Microsoft.Azure.Devices.Client.Samples
+{
+    /// <summary>
+    /// Parameters for the application.
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            'i',
+            "IotHubConnectionString",
+            Required = true,
+            HelpText = "The primary connection string for the IoT Hub instance to connect to.")]
+        public string IotHubConnectionString { get; set; }
+
+        [Option(
+            's',

```

Comment:
In other samples, we used 's' for secondary hub connection string. Although we don't use that here, it is a potential conflict to be aware of.

I think it is fine to use 's' here but I wanted you to be aware.